freud/cluster.pyx

```diff
@@ -206,45 +207,122 @@ cdef class ClusterProperties(_Compute):
-        """(:math:`N_{clusters}`, 3) :class:`numpy.ndarray`: The centers of
-        mass of the clusters."""
+        r"""(:math:`N_{clusters}`, 3) :class:`numpy.ndarray`: The geometric centers
+        of the clusters, independent of mass and defined as
+
+        .. math::
+
+            \mathbf{C}_m = \frac{1}{N} \sum_{i=0}^{N} \mathbf{r_i}
+
+        where :math:`N` is the number of particles and :math:`\mathbf{r_i}` are
+        their positions.
+        """
         return freud.util.make_managed_numpy_array(
             &self.thisptr.getClusterCenters(),
             freud.util.arr_type_t.FLOAT, 3)
 
+    @_Compute._computed_property
+    def centers_of_mass(self):
+        r"""(:math:`N_{clusters}`, 3) :class:`numpy.ndarray`: The centers of mass
+        of the clusters:
+
+        .. math::
+
+            \mathbf{C}_m = \frac{1}{M} \sum_{i=0}^{N} m_i \mathbf{r_i}
+
+        where :math:`M` is the total mass of particles, :math:`\mathbf{r_i}` are
+        their positions and :math:`m_i` are their masses.
+        """
+        return freud.util.make_managed_numpy_array(
+            &self.thisptr.getClusterCentersOfMass(),
+            freud.util.arr_type_t.FLOAT, 3)
+
     @_Compute._computed_property
     def gyrations(self):
-        """(:math:`N_{clusters}`, 3, 3) :class:`numpy.ndarray`: The gyration
-        tensors of the clusters."""
+        r"""(:math:`N_{clusters}`, 3, 3) :class:`numpy.ndarray`: The gyration
+        tensors of the clusters. Normalized by particle number:
+
+        .. math::
+
+            \mathbf{S} = \frac{1}{N} \begin{bmatrix}
+            \sum_i x_i^2    & \sum_i x_i y_i  & \sum_i x_i z_i \\
+            \sum_i y_i x_i  & \sum_i y_i^2    & \sum_i y_i z_i \\
+            \sum_i z_i x_i  & \sum_i z_i y_i  & \sum_i z_i^2   \\
+            \end{bmatrix}
+        """
         return freud.util.make_managed_numpy_array(
             &self.thisptr.getClusterGyrations(),
             freud.util.arr_type_t.FLOAT)
 
     @_Compute._computed_property
-    def radii_of_gyration(self):
-        """(:math:`N_{clusters}`,) :class:`numpy.ndarray`: The radius of
-        gyration of each cluster."""
-        return np.sqrt(np.trace(self.gyrations, axis1=-2, axis2=-1))
+    def inertia_tensors(self):
+        r"""(:math:`N_{clusters}`, 3, 3) :class:`numpy.ndarray`: The inertia
+        tensors of the clusters. Neither normalized by mass nor number:
+
+        .. math::
+
+            \mathbf{I} = \begin{bmatrix}
+            \sum_i m_i(y_i^2+z_i^2)& \sum_i -m_i(x_iy_i)& \sum_i -m_i(x_iz_i)\\
+            \sum_i -m_i(y_ix_i)& \sum_i m_i(x_i^2+z_i^2)& \sum_i -m_i(y_iz_i)\\
+            \sum_i -m_i(z_ix_i)& \sum_i -m_i(z_iy_i)& \sum_i (y_i^2+x_i^2)\\
+            \end{bmatrix}
+        """
+        return freud.util.make_managed_numpy_array(
+            &self.thisptr.getClusterMomentsOfInertia(),
+            freud.util.arr_type_t.FLOAT)
 
     @_Compute._computed_property
-    def sizes(self):
+    def cluster_sizes(self):
```

Comment:
There are significant implications for changing the name of properties. It forces us to bump our major version and the changes on this PR would not be able to be released until we release version 3.0.0. For now, please do not change property names.
```suggestion
    def sizes(self):
```

Reply:
If you feel the proposed name change is more appropriate, please make an issue.